Makefile

```
@@ -95,8 +95,7 @@ fmt:
 	find . -name '*.go' -not -path "./vendor/*" | xargs gofmt -w
 
 # Update dependencies
-vendor: Gopkg.toml
-	dep ensure -v
+vendor:	go mod vendor
```

```suggestion
vendor:
    go mod tidy
    go mod vendor
    go mod verify
```

We may want to add the verbose option (`-v`) to all of those commands too.